openedx/adg/lms/applications/models.py

```diff
@@ -119,7 +123,7 @@ class UserApplication(TimeStampedModel):
         validators=[FileExtensionValidator(['pdf', 'doc', 'jpg', 'png'])],
         help_text=_('Accepted extensions: .pdf, .doc, .jpg, .png'),
     )
-    cover_letter = models.TextField(blank=True, verbose_name=_('Cover Letter'), )
+    cover_letter_text = models.TextField(blank=True, verbose_name=_('Cover Letter'), )
 
```

Comment:
Is it necessary to change the field name as it might create conflicts with other branches?

Reply:
You're right. I realized this when I reviewed LP-2378. The `cover_letter` field is used in many instances in 2378, both in platform and theme. So I reverted this name change, made a new migration and updated all instances of `cover_letter_text` to `cover_letter` in my code.